handlers.go

```diff
@@ -203,6 +225,8 @@ func (a *autographer) handleSignature(w http.ResponseWriter, r *http.Request) {
 			X5U:        requestedSignerConfig.X5U,
 			SignerOpts: requestedSignerConfig.SignerOpts,
 		}
+		sigresps[i].X5U = rewriteLocalX5U(r, sigresps[i])
```

Comment:
nit: this could be done in the struct set up but no more review cycles required to do that